<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="tree1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>561</width>
     <height>401</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Equip. ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Function Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_equip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Equip 추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_ip">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_port">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>502</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PORT :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_equipname">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>명칭 :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_tag">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tag 추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_tagname">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>명칭 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_mbaddr">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>40001</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Address :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>오브젝트 리스트 저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>오브젝트 리스트 불러오기</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tree1</tabstop>
  <tabstop>add_equip</tabstop>
  <tabstop>input_equipname</tabstop>
  <tabstop>input_ip</tabstop>
  <tabstop>input_port</tabstop>
  <tabstop>add_tag</tabstop>
  <tabstop>input_tagname</tabstop>
  <tabstop>input_mbaddr</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
